src/components/modal/addtostudio/studio-button.jsx

```diff
@@ -50,11 +58,11 @@ const StudioButton = ({
             <div
                 className={classNames(
                     'studio-status-icon',
-                    {'studio-status-icon-unselected': !includesProject}
+                    {'studio-status-icon-unselected': !includesProject && !hasRequestOutstanding}
                 )}
             >
                 {(hasRequestOutstanding ?
-                    (<Spinner mode="smooth" />) :
+                    spinner :
```

Comment:
It still seems like this could keep being defined in place so it's easier to know what's happening there. The `spinner` variable is only used once, right here.

Reply:
hmmmm I will look into it thanks